tests/pipeline/test_adjusted_array.py

```diff
@@ -23,9 +24,21 @@
 from zipline.lib.adjusted_array import AdjustedArray, NOMASK
 from zipline.utils.numpy_utils import (
     datetime64ns_dtype,
+    default_missing_value_for_dtype,
     float64_dtype,
+    int64_dtype,
     make_datetime64ns,
 )
+from zipline.utils.test_utils import check_arrays, parameter_space
+
+
+def moving_window(array, nrows):
```

Comment:
I don't think we need to.  Anything that needs to do a moving window efficiently probably wants to use AdjustedArray.  This was just a simple utility for testing.